docs/docs/deployment/hosting/locally-frontend.md

```diff
@@ -101,6 +101,11 @@ Current variables used between 'frontend/environment.js' and 'frontend/common/pr
   Further reading on this value is available [here](https://web.dev/articles/samesite-cookies-explained). Default:
   'none'.
 
+### GitHub Integration Environment Variables
+
+- `GITHUB_APP_URL`: You can obtain the URL of your GitHub App in the 'About' section -> 'public link' and append
```

Comment:
Let's add an example here. It's always a bit confusing when you tell people to append things to a URL. I think it would be useful to give an example.

Reply:
Done